1-js/05-data-types/03-string/article.md

```diff
@@ -598,33 +598,33 @@ alert( '𝒳'.charCodeAt(0).toString(16) ); // d835, between 0xd800 and 0xdbff
 alert( '𝒳'.charCodeAt(1).toString(16) ); // dcb3, between 0xdc00 and 0xdfff
 ```
 
-You will find more ways to deal with surrogate pairs later in the chapter <info:iterable>. Probably, there are special libraries for that too, but nothing famous enough to suggest here.
+You will find more ways to deal with surrogate pairs later in the chapter <info:iterable>. There are probably special libraries for that too, but nothing famous enough to suggest here.
 
 ### Diacritical marks and normalization
 
-In many languages there are symbols that are composed of the base character and a mark above/under it.
+In many languages there are symbols that are composed of the base character with a mark above/under it.
 
-For instance, letter `a` can be the base character for: `àáâäãåā`. Most common "composite" character have their own code in the UTF-16 table. But not all of them, because there are too many possible combinations.
+For instance, the letter `a` can be the base character for: `àáâäãåā`. Most common "composite" character have their own code in the UTF-16 table. But not all of them, because there are too many possible combinations.
 
-To support arbitrary compositions, UTF-16 allows to use several unicode characters. The base character and one or many "mark" characters that "decorate" it.
+To support arbitrary compositions, UTF-16 allows us to use several unicode characters. The base character and one or many "mark" characters that "decorate" it.
 
 For instance, if we have `S` followed by the special "dot above" character (code `\u0307`), it is shown as Ṡ.
 
 ```js run
 alert( 'S\u0307' ); // Ṡ
 ```
 
-If we need a one more mark over the letter (or below it) -- no problem, just add the necessary mark character.
+If we need an additional mark above the letter (or below it) -- no problem, just add the necessary mark character.
 
 For instance, if we append a character "dot below" (code `\u0323`), then we'll have "S with dots above and below": `Ṩ`.
 
-The example:
+For example:
 
 ```js run
 alert( 'S\u0307\u0323' ); // Ṩ
 ```
 
-This gives great flexibility, but also an interesting problem: the same symbol visually can be represented with different unicode compositions.
+This provides great flexibility, but also an interesting problem: the same symbol can be visually represented with different unicode compositions.
```

Comment:
Sorry for my bad English, actually the idea was that different unicode compositions may represent the same visual character (see the example below).

The word "visually" referred to "the same symbol", not to "unicode composition".

Reply:
If the word 'visually' is needed to let the reader understand that the symbol _looks_ the same but in the unicode it's actually _different_, then maybe use:

'this same symbol can be represented with different unicode compositions, but still visually look the same'.

Reply:
"This provides great flexibility, but also an interesting problem: two characters may visually look the same, but be represented with different unicode compositions." - ok?

Reply:
Even better!

Reply:
please update the PR then, so that I can accept it along with other changes :)